examples/us-dams/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf src/.observablehq/cache",
+    "build": "rimraf dist && observable build",
+    "dev": "observable preview",
+    "deploy": "observable deploy",
+    "observable": "observable"
+  },
+  "dependencies": {
+    "@observablehq/framework": "latest",
+    "d3-dsv": "^3.0.1",
+    "d3-time-format": "^4.1.0",
+    "fs": "^0.0.1-security",
```

Comment:
```suggestion
```

You don’t need to install `fs`. It’s built-in to node. (And thankfully npm reserved it against malicious use!)